### ISSUE TITLE
consistency in terms on interface e..g observed

### ISSUE THREAD

**Issue by JDWoodcock** [npct/pct-shiny#126]
Sometimes e.g. flows it says cycle commutes & cycle commutes (scenario). This is okay although better would be 'cycle commuters (Census 2011)' and 'cycle commuters (scenario)'.

On route network it says 'observed' - it should say 'Census 2011' not observed (pedantically it is reported not observed). Also suggest when looking at baseline we don't need to see same no. twice- once baseline, once scenario.

At area level it says 'cyclists (scenario)' - this should be 'cycle commutes (scenario)' and we should add baseline 'cycle commutes (Census 2011)'. 

So in summary - always say Cycle commuters- not commutes or cyclists. Dont say 'observed'. In a change scenario give both baseline and change. For baseline dont repeat numbers by giving baseline and baseline as scenario.

**Comment by audev** [comment]
@JDWoodcock  @Robinlovelace    Just to clarify. Re the %who drive (last week) vs. Car drive commutes (now), I didn't change the labels so I assume you guys decided to change it. 
Also it's different in Liverpool to the rest of country (I guess it's planned to be consistent).

**Comment by JDWoodcock** [comment]
@Robinlovelace looking at Kent for a Census 20011 it says.

Note the error in increase in commutes (81 instead of 0)
 If possible I would remove the lines Cycle commutes (scenario) and Increase in commutes - as they add no information. 

Total commutes 	1030
Cycle commutes 	81 (8%)
Cycle commutes (scenario) 	81
Increase in commutes   	81
Route Distance 	2.4 km
Av. Route Time 	20.7 min
Hilliness (average gradient) 	2.4%
Route Type 	Direct

**Comment by Robinlovelace** [comment]
Hi @JDWoodcock yes I noticed that - believe it was introduced here: https://github.com/npct/pct-shiny/commit/6be5d3d5a15beee41c135764febe466b51645336 @nikolai-b can you take a look at this please?

Happy for @audev to tackle this also if up for it. I could also take a look if needs be.

Cheers for the testing @JDWoodcock

**Comment by audev** [comment]
@Robinlovelace  If the decision is clear and done happy to deal with it myself.

**Comment by Robinlovelace** [comment]
Yes I'm happy for this to be done - assigned to you for now, any issues let me know.

**Comment by Robinlovelace** [comment]
Just to add - for ease of coding @audev I think it's sufficient to for it to say:

Increase in commutes 0

In the above example instead of 81.

Consistency is good!

**Comment by JDWoodcock** [comment]
@audev and if you are unsure what wording is helpful on the earlier points please ask me. 
@Robinlovelace I prefer to get rid of non informative lines but not very strongly.
But we do need consistency in our use  of terms like observed & scenario.

I have also noticed at area level
Zone: 	Ashford 002
Cyclists (scenario): 	51.17
% who drive:   	81
Hilliness:   	2.34°

We don't want to have one as absolute and one as %. Suggest put absolute no. of drivers and total no. of commuters.

**Comment by nikolai-b** [comment]
@Robinlovelace regarding https://github.com/npct/pct-shiny/commit/6be5d3d5a15beee41c135764febe466b51645336#diff-32517cfb3ffe7234dab5f31a638f6028L99 sorry I was too hasty.  `tableOLC` should be brought back but that giant `if else` with the only difference being the table wasn't ideal.

**Comment by nikolai-b** [comment]
To clarify, the `if else` on
https://github.com/npct/pct-shiny/commit/6be5d3d5a15beee41c135764febe466b51645336#diff-32517cfb3ffe7234dab5f31a638f6028L116
and
https://github.com/npct/pct-shiny/commit/6be5d3d5a15beee41c135764febe466b51645336#diff-32517cfb3ffe7234dab5f31a638f6028L148

**Comment by Robinlovelace** [comment]
@audev make sense - you up for that? Good way to get your head round the code. Cheers for the clarification @nikolai-b.

**Comment by audev** [comment]
I'll start looking into this today. Any doubts on this or other issues, on Friday's meeting.

**Comment by nikolai-b** [comment]
Moved to #140
